GitLink.sublime-commands

```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "caption": "GitLink: Copy Git URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false
+        }
+    },
+    {
+        "caption": "GitLink: Open Git URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false,
+            "web": true
+        }
+    },
+    {
+        "caption": "GitLink: Copy Git URL to line",
+        "command": "gitlink",
+        "args": {
+            "line": true
+        }
+    },
+    {
+        "caption": "GitLink: Open Git URL to line",
+        "command": "gitlink",
+        "args": {
+            "line": true
+        }
+    },
+    {
+        "caption": "GitLink: Copy Git Blame URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false,
+            "blame": true
+        }
+    },
+    {
+        "caption": "GitLink: Open Git Blame URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false,
+            "web": true,
+            "blame": true
+        }
+    }
```

Comment:
I think I have all of the possible combinations, but do you know what the most common shortcuts are? We could add only the "Open" variations and add instructions to the README for how users can customize the command palette (add snippets in a new `GitLink.sublime-commands` in their `User/` directory)

Reply:
I imagine we'd also want menu items for navigating directly to the line as well, no?

Reply:
Done! https://github.com/rscherf/GitLink/pull/24/commits/1af1c185b3c446e8902d11dd4562f49c46c99964